client/plots/hierCluster.js

```diff
@@ -181,12 +181,21 @@ export class HierCluster extends Matrix {
 		// all parameters here must remove payload properties that are
 		// not relevant to the data request, so that the dofetch and/or
 		// browser caching would work
+		if (s.clusterMethod != 'average' && s.clusterMethod != 'complete') throw 'Invalid cluster method'
+		if (
```

Comment:
simplify. export an array from hierCluster.interactivity.js and check against that array